src/transform/benches/mzbench_opt/__init__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from enum import Enum
+
+
+class Scenario(Enum):
```

Comment:
Please ignore this, but instead of requiring code changes and updating the enum we could make things "data-driven" and dynamically find matching schemas + workloads or explicitly provide the two file paths.

Reply:
Done.